### DISCUSSION TITLE
Documentation License?

### DISCUSSION THREAD

**Discussion by squarewave17** [uikit/uikit#4535] [General]
Hello!

I'm using UIKit in a plugin i'm making. It's modified quite heavily but i'm still using a fair bit of original properties.

On the parts that remain the same, is it ok to use parts of the documentation, mark up etc to use in my documentation?

Thanks

**Comment by AlekVolsk** [comment]
https://github.com/uikit/uikit/blob/develop/LICENSE.md

**Reply by squarewave17** [reply]
Thanks, but it's not the UIKit license I'm struggling with..... What I'm basically asking, is can I include parts of the documentation in my own?

**Reply by AlekVolsk** [reply]
can you

**Reply by squarewave17** [reply]
?
